public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -74,43 +73,42 @@ block quickstart-repo
 .l-sub-section
   :marked
     More precisely, the redisplay occurs after some kind of asynchronous event related to
-    the view such as a keystroke, a timer completion, or an async `XHR` response.
-    We don't have those in this sample.
-    But then the properties aren't changing on their own either. For the moment we must operate on faith.
+    the view, such as a keystroke, a timer completion, or an async `XHR` response.
 
 :marked
-  Notice that we haven't called **new** to create an instance of the `AppComponent` class.
-  Angular is creating an instance for us. How?
+  Notice that you don't call **new** to create an instance of the `AppComponent` class.
+  Angular is creating an instance for you. How?
 
-  Notice the CSS `selector` in the `@Component` !{_decorator} that specifies an element named `my-app`.
-  Remember back in [QuickStart](../quickstart.html) that we added the `<my-app>` element to the body of our `index.html` file:
+  The CSS `selector` in the `@Component` !{_decorator} specifies an element named `my-app`.
+  Remember back in [QuickStart](../quickstart.html) that you added the `<my-app>`
+  element to the body of your `index.html` file:
 
 +makeExcerpt('index.html', 'body')
 
 :marked
-  When we bootstrap with the `AppComponent` class (in <ngio-ex path="main.ts"></ngio-ex>), Angular looks for a `<my-app>`
+  When you bootstrap with the `AppComponent` class (in <ngio-ex path="main.ts"></ngio-ex>), Angular looks for a `<my-app>`
   in the `index.html`, finds it, instantiates an instance of `AppComponent`, and renders it
   inside the `<my-app>` tag.
 
-  Try running the app. It should display the title and hero name:
+  Now run the app. It should display the title and hero name:
 figure.image-display
   img(src="/resources/images/devguide/displaying-data/title-and-hero.png" alt="Title and Hero")
 
 +ifDocsFor('ts')
   :marked
-    Let's review some of the choices we made and consider alternatives.
+    Review some of the choices you made and consider alternatives.
 
 :marked
   ## Template inline or template file?
 
-  We can store our component's template in one of two places.
-  We can define it *inline* using the `template` property, as we do here.
-  Or we can define the template in a separate HTML file and link to it in
+  You can store your component's template in one of two places.
+  You can define it *inline* using the `template` property, or you can define
+  the template in a separate HTML file and link to it in
   the component metadata using the `@Component` !{_decorator}'s `templateUrl` property.
 
   The choice between inline and separate HTML is a matter of taste,
   circumstances, and organization policy.
-  Here we're using inline HTML because the template is small, and the demo
+  Here you're using inline HTML because the template is small and the demo
```

Comment:
Used "the app uses". Couldn't think of anything clearer. Will revisit if I do, though.